lib/components/viewers/trip-viewer.js

```diff
@@ -187,8 +195,8 @@ class TripViewer extends Component {
                 }
 
                 return (
-                  <Stop aria-label={stopLabel} key={i} title={stopLabel}>
-                    <FlexWrapper width>
+                  <Stop aria-label={stopLabel} key={i}>
+                    <FlexWrapper width="80%">
```

Comment:
I think I'd almost prefer an inline style here rather than this extra prop, but will leave it to the second reviewer to decide!

Reply:
I favor inlining the `80%` right into the CSS.